doc/source/raysgd/raysgd_ref.rst

```diff
@@ -26,6 +26,21 @@ BaseTorchTrainable
     :members:
     :private-members:
 
+
+.. _tune-sgd-ddp-doc:
+
+Ray Tune + RaySGD
+-----------------
```

Comment:
should we move this into Tune?

Reply:
It might make sense to at least crossreference this in the Tune docs.